packages/neos-ui-redux-store/src/CR/Nodes/selectors.js

```diff
@@ -25,13 +36,25 @@ export const nodeByContextPath = state => contextPath => {
     return node;
 };
 
+export const makeGetDocumentNodes = nodeTypesRegistry => createSelector(
+    [
+        nodes
+    ],
+    nodesMap => {
+        const documentSubNodeTypes = nodeTypesRegistry.getSubTypesOf('TYPO3.Neos:Document');
+        const nodes = nodesMap.toJS();
```

Comment:
meh: we found out, that `toJS` at this point leads to performance issues, since it always results in different object references, while Immutable itself leaves references intact. So we should try to avoid `toJS` it selectors.